.credo.exs

```diff
@@ -16,7 +16,10 @@
       parse_timeout: 5000,
       color: true,
       checks: %{
-        disabled: []
+        disabled: [],
+        enabled: [
+          {Credo.Check.Consistency.UnusedVariableNames, []}
+        ]
```

Comment:
We cannot do this as this will result in only adding the `Credo.Check.Consistency.UnusedVariableNames` check.

> ❯ mix credo
Checking 51 source files ...
Analysis took 0.06 seconds (0.06s to load, 0.00s running 1 check on 51 files)

Note that the log displays `1 check` only.

The reason is that credo automatically adds all default checks if the `enabled` key is not defined. If its there, it will use the configured checks only.

Instead of:

```elixir
checks: %{
  disabled: [],
  enabled: [
    {Credo.Check.Consistency.UnusedVariableNames, []}
  ]
}
```

we can do this

```elixir
checks: [
  {Credo.Check.Consistency.UnusedVariableNames, []}
]
```

Reply:
I updated this so now there is: `Analysis took 1.8 seconds (0.05s to load, 1.7s running 67 checks on 52 files)`